src/containers/ServiceAccounts/ServiceAccounts.js

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import isEqual from 'lodash.isequal';
+// import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
+// import { getErrorMessage, urls } from '@tektoncd/dashboard-utils';
```

Comment:
guessing these are left in place for the follow-up PR linking to the details page?

Reply:
yes!